src/clr-angular/popover/dropdown/providers/dropdown-focus-handler.service.ts

```diff
@@ -16,6 +16,7 @@ import { FocusService } from '../../../utils/focus/focus.service';
 import { FocusableItem } from '../../../utils/focus/focusable-item/focusable-item';
 import { linkParent, linkVertical } from '../../../utils/focus/focusable-item/linkers';
 import { wrapObservable } from '../../../utils/focus/wrap-observable';
+import { take } from 'rxjs/internal/operators';
```

Comment:
Should this just be importing from `rxjs/operators` ?

Reply:
Thanks for catching that. Updated.